src/test/java/pl/techlab24/OSKManager/model/validation/RideDetailsValidatorTest.java

```diff
@@ -81,4 +97,19 @@ void shouldValidateRideDuration(BigDecimal rideDuration, List<String> expected)
             Arguments.of(BigDecimal.valueOf(-1L), Collections.singletonList("Ride duration cannot be lower than 0."))
         );
     }
+
+    @Test
+    void shouldValidateCourseClientMethodCallValidateMethodFromCourseClientValidatorClass() {
```

Comment:
Can we consider renaming methods in test classes. I might suggest separating the words with an underscore to make these names easier to read. For example -"should_Validate_Course_Client_Method_Call_Validate_Method_From_Course_Client_Validator_Class()". Or is it a bad practise?

Reply:
To check it try to find in Google: can underscore be used in java method name
And: https://stackoverflow.com/questions/23582541/why-java-method-name-should-not-contain-underscore?rq=1

Reply:
Maybe we can use @DisplayName annotation?
https://junit.org/junit5/docs/current/user-guide/#writing-tests-display-names